packages/bpk-stylesheets/base.js

```diff
@@ -59,58 +79,34 @@
-    return;
-  }
-
-  var classNames = [];
-
-  // touch support
-  classNames.push('ontouchstart' in window || window.DocumentTouch && document instanceof DocumentTouch // eslint-disable-line
-  ? 'touch-support' : 'no-touch-support');
-
-  // add more feature tests here...
-
-  document.documentElement.className += ' ' + classNames.map(function (className) {
-    return 'bpk-' + className;
-  }).join(' ');
-})(); /*
-       * Backpack - Skyscanner's Design System
-       *
-       * Copyright 2016-2019 Skyscanner Ltd
-       *
-       * Licensed under the Apache License, Version 2.0 (the "License");
-       * you may not use this file except in compliance with the License.
-       * You may obtain a copy of the License at
-       *
-       *   http://www.apache.org/licenses/LICENSE-2.0
-       *
-       * Unless required by applicable law or agreed to in writing, software
-       * distributed under the License is distributed on an "AS IS" BASIS,
-       * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-       * See the License for the specific language governing permissions and
-       * limitations under the License.
-       */
+eval("\n\n__webpack_require__(/*! ./index.scss */ \"./index.scss\");\n\n(function () {\n  if (typeof document === 'undefined') {\n    return;\n  }\n\n  var classNames = [];\n\n  // touch support\n  classNames.push('ontouchstart' in window || window.DocumentTouch && document instanceof DocumentTouch // eslint-disable-line\n  ? 'touch-support' : 'no-touch-support');\n\n  // add more feature tests here...\n\n  document.documentElement.className += ' ' + classNames.map(function (className) {\n    return 'bpk-' + className;\n  }).join(' ');\n})(); /*\n       * Backpack - Skyscanner's Design System\n       *\n       * Copyright 2016-2019 Skyscanner Ltd\n       *\n       * Licensed under the Apache License, Version 2.0 (the \"License\");\n       * you may not use this file except in compliance with the License.\n       * You may obtain a copy of the License at\n       *\n       *   http://www.apache.org/licenses/LICENSE-2.0\n       *\n       * Unless required by applicable law or agreed to in writing, software\n       * distributed under the License is distributed on an \"AS IS\" BASIS,\n       * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n       * See the License for the specific language governing permissions and\n       * limitations under the License.\n       */\n\n//# sourceURL=webpack:///./index.js?");
 
 /***/ }),
-/* 1 */
-/***/ (function(module, exports) {
 
-// removed by extract-text-webpack-plugin
+/***/ "./index.scss":
+/*!********************!*\
+  !*** ./index.scss ***!
+  \********************/
+/*! no static exports found */
+/***/ (function(module, exports, __webpack_require__) {
+
+eval("// extracted by mini-css-extract-plugin\n\n//# sourceURL=webpack:///./index.scss?");
 
 /***/ })
-/******/ ]);
\ No newline at end of file
+
```

Comment:
Always a bit nervous when the base stylesheet changes, do we have a good way to test nothing is broken? cc @matthewdavidson

Reply:
The only functionality of the javascript part of the base stylesheet is to append `bpk-no-touch-support` or ` bpk-touch-support` based on the browsers ability to support touch interactions. If hovers styles still work then 👍

Reply:
@shaundon - This was based on your changes that you had made when we moved from webpack version upgrade

Reply:
Will test using the recommendation from @matthewdavidson above

Reply:
Thanks both 😄

Reply:
Works as normal when doing the recommendation above 😃